.github/workflows/release-nightly.yml

```diff
@@ -3,8 +3,8 @@ name: Release Nightly
 on:
   workflow_dispatch:
   schedule:
-    # 08:00 AM Beijing Time. Except Tuesday, which is for full release
-    - cron: "0 0 * * 0,1,3,4,5,6"
+    # 08:00 AM Beijing Time on everyday
```

Comment:
maybe we can trigger early so that we can get all tests result at 10:00 am

Reply:
eco-system tests are tests, please run them in-sync with the rest of the test suite, instead of running out-of-sync.

Let's fix the root problem of running tests instead of changing the process.